engine/render/src/dmsdk/render/render.h

```diff
@@ -224,13 +229,15 @@ namespace dmRender
     struct RenderListEntry
     {
         dmVMath::Point3 m_WorldPosition;
+        uint64_t m_UserData; // E.g. component type instance pointer
         uint32_t m_Order;
         uint32_t m_BatchKey;
         uint32_t m_TagListKey;
-        uint64_t m_UserData;
-        uint32_t m_MinorOrder:4;
-        uint32_t m_MajorOrder:2;
-        uint32_t m_Dispatch:8;
+        uint32_t m_MinorOrder : 4;
+        uint32_t m_MajorOrder : 2;
+        uint32_t m_Dispatch   : 8;
+        uint32_t m_Visibility : 1; // See enum Visibility
+        uint32_t              : 17;
     };
```

Comment:
The size changed form 48 bytes to 40, by simply sorting the elements by size, and adhere to the padding rules of C++.